### DISCUSSION TITLE
Add the progress bar to `update` command

### DISCUSSION THREAD

**Discussion by tjapro** [tldr-pages/tldr#10213] [Ideas]
Today, when `tldr` runs `update` or we run `tldr --update` there are not any feedback about what is happening.  
The only feedback is in the end: "Successfully updated cache.".  

The idea for a feature is to add a progress bar while the update is running.

Also, a parallel idea: make the auto-update running in background, while is showing the actual command `tldr` page.

**Comment by kbdharun** [comment]
> Also, a parallel idea: make the auto-update running in background, while is showing the actual command `tldr` page.
> 

It does run in the background, can you be more specific with what client you are using?

**Reply by tjapro** [reply]
I'm using `tealdeer`.

**Reply by kbdharun** [reply]
> I'm using `tealdeer`.

Hi, I just now checked the `tealdeer` repo, [you need to enable auto updating the cache manually](https://github.com/dbrgn/tealdeer/issues/250) (And it seems to be the intended behaviour by default) by following the [docs](https://dbrgn.github.io/tealdeer/config.html) [here](https://dbrgn.github.io/tealdeer/config_updates.html) you can enable it in your system.
